service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/selector/ChildSelector.java

```diff
@@ -16,14 +19,25 @@
     private String childCount = "";
     @Builder.Default
     private List<Integer> selected = new ArrayList<>();
+    @Builder.Default
+    private List<Integer> hidden = new ArrayList<>();
 
-    public void generateChildCount(int max) {
+    public static String generateChildCount(int max) {
```

Comment:
Is there a reason for this to return something now? As well as the method that you added.

Reply:
It was previously a void method with a side effect of setting the childCount. I thought it would be cleaner to have a method with a return value.

Reply:
I think it would be nice to, maybe rename it to setChildCountFromInt or something like that and make sure it is annotated with a json ignore

Reply:
Same for the new method you added, `setHiddenFromChildList` or something like that